middleware-jackson/src/main/java/com/nytimes/android/external/store/middleware/jackson/JacksonStringParser.java

```diff
@@ -29,10 +30,10 @@ public JacksonStringParser(@Nonnull ObjectMapper objectMapper, @Nonnull Type typ
     }
 
     @Override
-    @Nullable
-    public Parsed call(@Nonnull String source) {
+    @SuppressWarnings({"PMD.EmptyCatchBlock"})
```

Comment:
yup makes sense. Store drops errors from disk read currently since there is network to fallback to.  If parsing error occurs from network it makes sense to throw error instead of overwriting the previous cached value with a null